docs/application/native/guides/multimedia/sound-manager.md

```diff
@@ -464,6 +468,74 @@ To add an external USB device for stream routing:
    sound_manager_free_device_list(device_list);
    ```
 
+<a name="stream_effect_method"></a>
+## Adding effect methods to a stream information
+
+There are couple of effect methods for preprocessing. When you start to capture audio streams, you can enable several methods simultaneously.
+If these methods are set, the capture stream will be preprocessed by the audio framework before receiving it.
+There are two types of APIs that you can use.
+
+- In case of exclusive methods named `sound_effect_method_with_reference_e`, It includes Acoustic Echo Cancellation(AEC) functionality and it can not be set with other methods because it uses a playback reference that is a exclusive system resource.
+- In case of `sound_effect_method_e`, There are like Noise-Suppression(NS) and Automatic Gain Control(AGC) that don't need a reference playback. Therefore, you can use these methods simultaneously using 'or' keyword.
```

Comment:
```suggestion
- In case of `sound_effect_method_e`, there are functionalities like Noise-Suppression (NS) and Automatic Gain Control (AGC) that don't need a reference playback. Therefore, you can use these methods simultaneously using the 'or' keyword.
```